packages/openneuro-client/src/client.js

```diff
@@ -69,7 +69,43 @@ const middlewareAuthLink = (uri, getAuthorization, fetch) => {
   }
 }
 
-const createLink = (uri, getAuthorization, fetch) => {
+const parse = version => version.split('.').map(n => parseInt(n))
+const checkVersions = (serverVersion, clientVersion) => {
+  const [serverMajor, serverMinor] = parse(serverVersion)
+  const [clientMajor, clientMinor] = parse(clientVersion)
+  if (serverMajor > clientMajor || serverMinor > clientMinor) {
+    console.warn(
+      `Your openNeuro client is out of date (v${clientVersion}). We strongly recommend you update to the latest version (v${serverVersion}) for an optimal experience.`,
+    )
+  } else if (
+    serverMajor < clientMajor ||
+    (serverMajor === clientMajor && serverMinor < clientMinor)
+  ) {
+    // panic, then
+    console.warn(
+      'Your openNeuro client is out of date. We strongly recommend you update to the most recent version for an optimal experience.',
+    )
+  }
+}
```

Comment:
For consistency with other version comparisons this should use the semver library we're already using elsewhere.